.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  bracketSpacing: false,
+  jsxBracketSameLine: true,
+  singleQuote: true,
```

Comment:
Most of the files are double quote, so should we leave this false? :)

Reply:
This also came with the upgrade. I'll change it to double